adoptadog-index.html

```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Adopt A Dog Home Page</title>
+    <link rel="stylesheet" type="text/css" href="styles/adoptadog-stylesheet.css">
+    <script src="https://kit.fontawesome.com/015d63ce65.js" crossorigin="anonymous"></script>
+    <!--must replace pipe (|) with %7C-->
+    <link href="https://fonts.googleapis.com/css?family=Courgette%7CPT+Sans:700&display=swap" rel="stylesheet">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+  </head>
+  <body>
+
+    <!-- Create header w/ background -->
+    <header>
+      <div class="grid-container">
+        <div class="grid-item-1">
+          <img src="images/adoptadog-logo.svg" alt="Adopt A Dog Logo" style="min-width:20rem; max-width:60%;">
+          <h1 class="header-h1">Seattle's premiere dog adoption agency</h1>
+        </div>
+        <div class="grid-item-2">
+          <img src="images/cart-icon.svg" alt="Click to go to checkout" style="min-width: 2rem; max-width:5%;">$1234.56
+          <p><a href="adoptadog-checkout.html"><img src="images/checkout-button.jpg" alt="Click to go to checkout" style="min-width: 7.5rem; max-width:30%;"></a>
+        </div>
+      </div>
+    </header>
+
+    <!-- Create navigation pane -->
+      <nav class="nav-header">
+        <a class="nav-link" href="adoptadog-index.html">Home / </a>
+        <a class="nav-link" href="adoptadog-dogs.html">Dogs / </a>
+        <a class="nav-link" href="adoptadog-blog.html">Blog</a>
+      </nav>
+
+      <!--Home intro-->
+      <div class="grid-container">
+        <div class= "grid-item-3">
+          <img src="images/home-intro.jpg" alt="Introduction photo of dog and his owner" style="width:100%;">
```

Comment:
The Image and Text look good at larger resolutions but as the window shrinks you will notice a lot of wasted white space under the image in relation to text on the right. You can use breakpoints to change the layout and address this as we will cover later. You also might consider keeping the image and text as siblings in the same parent div and change the styles to achieve a similar look. This will allow the text to wrap and take up the white space at smaller resolutions.